### ISSUE TITLE
Discrepancy with install-time modules when installing locally vs installing through play store

### ISSUE THREAD

**Issue by Lexxicon** [google/bundletool#176]
**Describe the bug**
The results of installing an app bundle onto a device results in an install-time module to be included, but when installing the same bundle onto the same device through the play store, that install-time module is not included.

**Bundletool version(s) affected**
Version: 1.1.0

**To Reproduce**
The results of executing the following commands has different behavior compared to installing the app via play store.
```
java -jar bundletool-all-1.1.0.jar build-apks --bundle=android-Primary-release.aab --output=android-Primary-release.apks
java -jar bundletool-all-1.1.0.jar install-apks --apks=android-Primary-release.apks  
```
An install-time module configured as shown below, will install onto the device when using 'build-apks' and 'install-apks' commands of the bundle tool, but when installing the same app onto the same device via play store, the module will not be included.
```
<dist:module
    dist:instant="false"
    dist:title="@string/title_gles32">
    <dist:fusing dist:include="false"/>
    <dist:delivery>
        <dist:install-time>
            <dist:conditions>
                <dist:device-feature dist:name="reqGlEsVersion=0x30002" />
            </dist:conditions>
        </dist:install-time>
    </dist:delivery>
</dist:module>
<application android:hasCode="false"/>
```
I have also tried the following variations
```
                <dist:conditions>
                    <dist:device-feature
                        dist:name="android.hardware.opengles.version"
                        dist:version="0x30002" />
                </dist:conditions>
```
```
                <dist:conditions>
                    <dist:device-feature
                        dist:name="android.hardware.opengles.version"
                        dist:version="0x00030002" />
                </dist:conditions>
```

**Expected behavior**
I expected the resulting installed app to be the same initial configuration if installed by the bundle tool or from the play store. 

**Known workaround**
none found yet

**Environment:**
OS: Windows 10
Device: LGUS 997 running Android 8.0.0

**Additional context**
My goal is to have a specific feature module be present at install time depending on the gles version present on the target device. The intent is to supply textures in a supported compression format for the specific device.

**Comment by ymakhno** [comment]
I tried to reproduce this issue with our samples: https://github.com/android/app-bundle-samples/tree/main/DynamicFeatures on my Pixel 3 and added conditional delivery for `features/java` module, included the following code into its AndroidManifest.xml:
```
<dist:conditions>
    <dist:device-feature	
      dist:name="android.hardware.opengles.version"	
      dist:version="0x00030002" />
</dist:conditions>
```

It works fine for me. If you still experience the issue could you please add link to a simple project that I can use to reproduce the issue?

**Comment by ymakhno** [comment]
Please disregard my previous comment I found a way to reproduce the issue.

**Comment by Lexxicon** [comment]
Glad to hear it. I am no longer on the project where we encountered the issue anymore. I'll try and forward this ticket onto the other engineers in case you need more context.

**Comment by ymakhno** [comment]
Fixed in 1.5.0.
